packages/next-frontend/components/hype-registration-button.tsx

```diff
@@ -55,16 +77,15 @@ export default function HypeRegistrationButton() {
 
 		const getBalance = async () => {
```

Comment:
There does seem to be an issue where the connected wallet is not being recognized every time upon refresh. The user should be able to log in once, and then every time after that they don't have to log in again,  until they clear local storage or it expires

Reply:
See https://github.com/HypeDAO/HypeDAO/pull/8/files#r701784895